lms/djangoapps/learner_dashboard/views.py

```diff
@@ -45,3 +45,20 @@ def view_programs(request):
     }
 
     return render_to_response('learner_dashboard/programs.html', context)
+
+
+@login_required
+@require_GET
+def program_details(request, programId):
```

Comment:
Pylint's also complaining because these variables are currently unused. Silence it for now by putting `# pylint: disable=unused-argument` at the end of this line. We'll remove it when this view is more fleshed-out.